coordinator/src/main/java/org/apache/uniffle/coordinator/web/servlet/NodesServlet.java

```diff
@@ -37,20 +37,36 @@ public NodesServlet(CoordinatorServer coordinator) {
   }
 
   @Override
-  protected Response<List<ServerNode>> handleGet(HttpServletRequest req, HttpServletResponse resp) {
-    List<ServerNode> serverList = coordinator.getClusterManager().getServerList(Collections.EMPTY_SET);
-    String id = req.getParameter("id");
-    String status = req.getParameter("status");
-    serverList = serverList.stream().filter((server) -> {
-      if (id != null && !id.equals(server.getId())) {
-        return false;
+  protected Response handleGet(HttpServletRequest req, HttpServletResponse resp) {
+    String requestURI = req.getRequestURI();
+    String methodName = requestURI.substring(requestURI.lastIndexOf("/"));
+    if (methodName == null || methodName.length() == 0) {
+      return Response.success("The requested address is incorrect, please confirm!");
+    } else {
+      if (methodName.equalsIgnoreCase("getUnhealthy")) {
```

Comment:
I mean if status=UNHEALTHY, get nodes from `UnhealthyServerList`, else if status=LOST, get nodes from `LostServerList`, otherwise, get nodes from `ServerList`

Reply:
I think this is a more RESTful style, and if you introduce a RESTful framework, it is easier to switch

Reply:
I mean we should unify the parameters, one parameter is clearer for users.
And if we do this, users need to understand the mapping between method names and status

Reply:
I see, but now I have changed it to a RESTful style, which I feel is more reasonable

Reply:
What is your opinion? @jerqi @advancedxy

Reply:
We can refer to the style of Yarn
![Uploading 企业微信截图_90117f9f-fd0d-4be5-8901-c9ee2373b510.png…]()

Reply:
I don't get your point. And i think yarn is not doing very well. @jerqi

Reply:
Your style is ok. The most important thing is that we should have consistent style. If you choose this style, we can use it.

Reply:
Currently, the url will be like /api/server/nodes?status=ACTIVE, but in this pr, it will be like /api/server/nodes/getActive?status=ACTIVE, i think it is weird. In the future, i want to change it to be like /api/server/nodes/ACTIVE.

Reply:
> Currently, the url is like /api/server/nodes?status=ACTIVE, but in this pr, it will be like /api/server/nodes/getActive?status=ACTIVE, i think it is weird. In the future, i want to change it to be like /api/server/nodes/ACTIVE.

Agree with you.

Reply:
I didn't understand what you meant just now, and I also agree with you that I remove the status, no need? status=ACTIVE This type@xianjingfeng

Reply:
You don't need remove status in this pr, i will do it in next pr. And you don't need add any parameters or change its url in this API,just use the existing parameters.